app/controllers/ops_controller.rb

```diff
@@ -570,7 +565,7 @@ def get_node_info(treenodeid, show_list = true)
     region_text = _("[Region: %{description} [%{region}]]") % {:description => MiqRegion.my_region.description,
                                                                :region      => MiqRegion.my_region.region}
     @right_cell_text ||= case x_active_tree
-                         when :diagnostics_tree then _("Diagnostics %{text}") % {:text => region_text}
+                         when :diagnostics_tree then _("Diagnostics %{text} 121212") % {:text => region_text} ## this is the header for the diagonists page (that is now empty) 
```

Comment:
typo: diagnostics

Reply:
whoop true true, although this comment isnt meant to stay here forever, its more a reminder so i can ask about what we want to do about this particular page

![Screen Shot 2022-04-22 at 3 29 48 PM](https://user-images.githubusercontent.com/29209973/164782466-3eda302f-4c8e-4c63-ab6a-453b55cff25b.png)

since now its basically an empty page, is there any note or comment or label we want to add here or do we want to simply leave it as is?

Reply:
If there's nothing to show, then we should remove the page.

Reply:
Are there even any Zone diagnostics after this?

Reply:
If you mean "is there anything left inside the zone diagnostics tab after this?" then the answer is yes
![Screen Shot 2022-04-25 at 11 02 12 AM](https://user-images.githubusercontent.com/29209973/165116736-553fcdad-b532-4c23-b3b5-91f44801ed83.png)

Reply:
Interesting - maybe not for this PR, but then I wonder if it's useful to even have the servers in the left side nav anymore, because you can get that same information in the servers tab there.  Then again, it's nice to be able to find a particular server if you don't know what zone it's in.

Reply:
Update for here: Just pushed up a change to finish emptying out the page (see [this](https://github.com/ManageIQ/manageiq-ui-classic/pull/8235#discussion_r856518476) screenshot) however i haven't been able to remove the 'clickyness' of the page and i don't think theres a way to specifically remove it since that functionality is tied to this method https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/javascript/oldjs/controllers/tree_view_controller.js#L101-L105 which looks to be pretty generic so i dont think we want to be adding diagnostics tree specific code there